packages/strapi-plugin-content-manager/admin/src/containers/ListView/index.js

```diff
@@ -125,15 +120,32 @@ function ListView({
 
   // Helpers
   const getMetaDatas = (path = []) =>
-    get(layouts, [slug, 'metadatas', ...path], {});
-  const getListLayout = () => get(layouts, [slug, 'layouts', 'list'], []);
+    get(layouts, [...contentTypePath, 'metadatas', ...path], {});
+
+  const getListLayout = () =>
+    get(layouts, [...contentTypePath, 'layouts', 'list'], []);
+
+  const getName = () => {
+    return getFeatureLabel(models, slug);
+  };
+
   const getAllLabels = () => {
     return sortBy(
       Object.keys(getMetaDatas())
         .filter(
           key =>
-            !['json', 'group', 'relation', 'richtext'].includes(
-              get(layouts, [slug, 'schema', 'attributes', key, 'type'], '')
+            ![
+              'json',
```

Comment:
I am seeing this a lot maybe we should create a utils to prevent code duplication. What do you think?